README.md

```diff
@@ -1,10 +1,15 @@
 # ![inspectIT](images/64x64.png?raw=true) inspectIT workshop
-The repository contains files needed for the inspectIT workshops. Please visit [official web-site](http://www.inspectit.rocks/) or [GitHub repository](https://github.com/inspectIT/inspectIT) for more information about inspectIT.
+The repository contains files needed for the inspectIT workshops. 
+Please visit [official web-site](http://www.inspectit.rocks/) or [GitHub repository](https://github.com/inspectIT/inspectIT) for more information about inspectIT.
+
+The inspectIT workshops uses the open source application "Spring Petclinic Microservices". The Pet Clinic application is a rather simple application based on the SpringBoot (Every microservice is a Spring Boot application) powered microservices. The original repository is located at github: https://github.com/spring-petclinic/spring-petclinic-microservices. All information that are required to understand this Application can be found here: https://speakerdeck.com/michaelisvy/spring-petclinic-sample-application
```

Comment:
I would not mention the original repo, seams not needed. For the second link I would wrap this URL in a link.